js/interface.js

```diff
@@ -1,3 +1,4 @@
+let saveButtonLabel = '';
```

Comment:
File doesn't look in line with ESLint standards.

Reply:
made update to incorporate ESlint standards